propnet/core/graph.py

```diff
@@ -174,11 +174,15 @@ def evaluate(self, material=None, property_type=None):
         Returns:
             void
```

Comment:
What is this `void`? :-)

Reply:
:)
I can remove it, but I'm not sure what is the right way to document that the method returns nothing.